js/common.mjs

```diff
@@ -0,0 +1,302 @@
+"use strict"
+//common layout functions used by both main.js and guide.js
+
+export {createLinkElement, initSingleCell, CELL_FORMAT_DISABLE_CHECKBOX, CELL_FORMAT_GUIDE, CELL_FORMAT_CHECKLIST, CELL_FORMAT_SKIP_ID, CELL_FORMAT_SHOW_CHECKBOX}
```

Comment:
Are we able to combine all the cell format data into a single format/Parameter object?